manifests/init.pp

```diff
@@ -95,6 +95,10 @@
 # @param [Array[Hash]] local_rules
 #   An array of hashes of rules to be added to /etc/falco/falco_rules.local.yaml
 #
+# @param [Boolean] watch_config_files
```

Comment:
please dont add the datatype. puppet strings will pick it up from the class signature.